core/src/test/java/com/expediagroup/streamplatform/streamregistry/core/events/NotificationEventEmitterConsumerBindingServiceTest.java

```diff
@@ -1,222 +0,0 @@
-/**
```

Comment:
why have all these tests been deleted?

Reply:
Because they massively overreach what they need to test, and don't really prove anything.

Reply:
I think completely deleting them is going too far.

Reply:
They also don't work. But thats because they are overreaching, not because the functionality under test is broken.

Reply:
I'm looking into restoring these tests, but they look more suited to an ITest scenario to me.

Reply:
Tests restored